### PR TITLE
tasks for Module 2

### PR DESCRIPTION
**Pull request by aliaksandrazaronak**
In this pull request I've added two functions: display_inventory and add_to_inventory.

# Pull Request Checklist

* [ ] I've checked code quality with flake8 and pylint tools
* [ ] I've checked compliance with Python docstring conventions with pydocstyle tool
* [ ] All third party packages are added to requirements.txt

### CODE REVIEW

File: tasks/Module_2/task_1.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+LIGHT_WEIGHT_THRESHOLD = 60
+HEAVY_WEIGHT_THRESHOLD = 70
+MAX_WEIGHT_THRESHOLD = 80
+
+def display_inventory(inventory):
+    print('\n'"Inventory:")
+    for item_name, item_count in inventory.items():
+        print(f"{item_count} {item_name}")
+
+    item_total = sum(stuff.values())
```
Comment (Pacu2) [inline tasks/Module_2/task_1.py:20]: You're accessing outer scope "stuff" here. Please use "inventory", as current implementation will fail if someone will run this function with a different argument.

File: tasks/Module_2/task_1.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+from collections import defaultdict
+
+stuff = {'rope': 2, 'torch': 6, 'gold coin': 42, 'dagger': 3, 'arrow': 12}
+
+dragonLoot = [
```
Comment (Pacu2) [inline tasks/Module_2/task_1.py:5]: can we please use `snake_case` naming convention here?

File: tasks/Module_2/task_1.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    'gold coin', 'chewed gum', 'dagger', 'used tissue', 'gold coin', 'chewed gum', 'torch',
+    'gold coin', 'rubbish', 'used tissue', 'chewed gum', 'dagger', 'used tissue']
+
+unused_stuff = ['rubbish', 'chewed gum', 'used tissue']
```
Comment (Pacu2) [inline tasks/Module_2/task_1.py:9]: I think it's more of `unwanted_stuff`, and I think it makes it more obvious why we skip those items, and chewed_gum sounds like it was used before

File: tasks/Module_2/task_1.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    if LIGHT_WEIGHT_THRESHOLD < item_total < HEAVY_WEIGHT_THRESHOLD:
+        print('\n'"CAUTION: Your backpack weighs a lot, your stamina runs out quicker!")
+    elif HEAVY_WEIGHT_THRESHOLD < item_total < MAX_WEIGHT_THRESHOLD:
+        print('\n'"CAUTION: Your equipment is very heavy, you're moving slower than usual!")
+    elif item_total >= MAX_WEIGHT_THRESHOLD:
+        print('\n'"CAUTION: You are overloaded, can't move!")
+
+def add_to_inventory(inventory, added_items):
+    skipped = defaultdict(int)
+    number_of_new_added_items = 0
```
Comment (Pacu2) [inline tasks/Module_2/task_1.py:31]: The `new` here might be misleading, as it's not new as it wasn't seen in the inventory before. 
I'd just name it `number_of_added_items` or `added_items_count` to keep it shorter.

File: tasks/Module_2/task_1.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    for new_item in added_items:
+        if new_item in unused_stuff:
+            skipped[new_item] += 1
+        else:
+            inventory[new_item] += 1
+            number_of_new_added_items +=1
+
+    print(f"Added {number_of_new_added_items} items to inventory")
+    print(f"Skipped:")
```
Comment (Pacu2) [inline tasks/Module_2/task_1.py:41]: No need for `f` here

File: tasks/Module_2/task_1.py [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+def display_inventory(inventory):
+    print('\n'"Inventory:")
+    for item_name, item_count in inventory.items():
+        print(f"{item_count} {item_name}")
+
+    item_total = sum(stuff.values())
+    print('\n'f"Total number of items: {item_total}")
+    if LIGHT_WEIGHT_THRESHOLD < item_total < HEAVY_WEIGHT_THRESHOLD:
+        print('\n'"CAUTION: Your backpack weighs a lot, your stamina runs out quicker!")
```
Comment (Pacu2) [inline tasks/Module_2/task_1.py:23]: I have mixed feelings about a single quote string followed with double-quoted string. Can we just wrap both of them with a double-quoted?

File: tasks/Module_2/task_1.py (hunk 1 above)
Comment (aliaksandrazaronak) [reply tasks/Module_2/task_1.py]: Done!

File: tasks/Module_2/task_1.py (hunk 2 above)
Comment (aliaksandrazaronak) [reply tasks/Module_2/task_1.py]: Converted to `snake_case`!

File: tasks/Module_2/task_1.py (hunk 3 above)
Comment (aliaksandrazaronak) [reply tasks/Module_2/task_1.py]: Renamed as proposed!

File: tasks/Module_2/task_1.py (hunk 4 above)
Comment (aliaksandrazaronak) [reply tasks/Module_2/task_1.py]: Replaced with `added_items_count`

File: tasks/Module_2/task_1.py (hunk 5 above)
Comment (aliaksandrazaronak) [reply tasks/Module_2/task_1.py]: Removed!

File: tasks/Module_2/task_1.py (hunk 6 above)
Comment (aliaksandrazaronak) [reply tasks/Module_2/task_1.py]: Moved under double-quoted string

File: tasks/Module_2/task_1.py [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+            added_items_count +=1
+
+    print(f"Added {added_items_count} items to inventory")
+    print("Skipped:")
+    for item_name, item_count in skipped.items():
+        print(f"{item_count} {item_name}")
+    return inventory
+
+new_inventory = add_to_inventory(stuff, dragon_loot)
+display_inventory(new_inventory)
```
Comment (Pacu2) [inline tasks/Module_2/task_1.py:47]: Please add empty line at the end of the file :)
It’s a good practice as it makes further git-diffs easier to read

Comment (aliaksandrazaronak) [reply tasks/Module_2/task_1.py]: Good spotting! Added!

**Pull request merged**